### DISCUSSION TITLE
Multiple PR Template selection

### DISCUSSION THREAD

**Discussion by john-opennet** [community/community#81319] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I have few templates for different purpose, such as `hotfix` / `bug` / `feature`, 
and I hope there is a dropdown to display `.md` template under `PULL_REQUEST_TEMPLATE` folder, 
instead of using `template=xxx.md` manually

**Comment by azure562** [comment]
It would be convenient if GitHub could pick up the pull request template at the folder level instead of the repo level.

So we can simply drop the different templates into their respective folder in the scenario described above.

**Comment by casperdcl** [comment]
Also there is currently no way to share a URL to open a PR using a custom template.

- works, but needs hard-coded branch: `https://github.com/OWNER/REPO/compare/main...FEAT?template=mytemplate.md`
- let user select feature branch, but clears template: `https://github.com/OWNER/REPO/compare/main?template=mytemplate.md` (i.e. redirects to `https://github.com/OWNER/REPO/compare/main...FEAT`)

desired: don't clear the `?template=mytemplate.md` when selecting a branch to compare to.

It already doesn't clear `?expand=1` so it should be trivial to implement using the same logic!

**Comment by Mel-iza** [comment]
It would be very interesting to be able to select the different types of templates! Or at least be able to use files as .yml files, just like issue templates - this way, certain information such as labels and assignees could be more customizable and updated automatically and we could also use custom forms.

Another option would be to allow us to have the `PULL_REQUEST_TEMPLATES/` folder with its respective files, such as /`bug.md` /`hotfix.md`, for example, yet still have a `pull_request_template.md` in `.github` that allows the calling or selection of these

**Reply by 0xdhrv** [reply]
Yeah, this would really help in lot use cases
